### PR TITLE
rc.1 blog post

### PR DESCRIPTION
**Pull request by krancour**
Signed-off-by: Kent Rancourt <kent.rancourt@microsoft.com>

### CODE REVIEW

File: content/2021-11-18-brigade-2-cuts-first-release-candidate.md
Comment (radu-matei) [inline content/2021-11-18-brigade-2-cuts-first-release-candidate.md:12]: Do you want to link to the actual release here?

File: content/2021-11-18-brigade-2-cuts-first-release-candidate.md
Comment (radu-matei) [inline content/2021-11-18-brigade-2-cuts-first-release-candidate.md:18]: nit: "peripherals" has been repeated a bunch of times already.

File: content/2021-11-18-brigade-2-cuts-first-release-candidate.md
Comment (radu-matei) [inline content/2021-11-18-brigade-2-cuts-first-release-candidate.md:20]: Do you want to link their GitHub / Twitter here?

**Review by radu-matei — APPROVED** [review]
LGTM with some tiny nits.

File: content/2021-11-18-brigade-2-cuts-first-release-candidate.md
Comment (krancour) [reply content/2021-11-18-brigade-2-cuts-first-release-candidate.md:20]: Good idea.

File: content/2021-11-18-brigade-2-cuts-first-release-candidate.md
Comment (krancour) [reply content/2021-11-18-brigade-2-cuts-first-release-candidate.md:12]: It seemed clunky to link to a chart, which is why I linked to the QuickStart.

File: content/2021-11-18-brigade-2-cuts-first-release-candidate.md
Comment (krancour) [reply content/2021-11-18-brigade-2-cuts-first-release-candidate.md:18]: Agree. There are three occurrences, which isn't _really_ a lot, but because it's a word we don't encounter a lot in daily use, seeing/hearing three times definitely does feel repetitive. I'll see if I can reword a little to avoid one use of that word.

**Comment by krancour** [comment]
@radu-matei thanks for the feedback -- I incorporated it into f3f933cc7ee3dde73748a82a7b41d98ba1c7e9d6.

File: content/2021-11-18-brigade-2-cuts-first-release-candidate.md
Comment (radu-matei) [reply content/2021-11-18-brigade-2-cuts-first-release-candidate.md:12]: I was thinking at the actual GitHub release -- https://github.com/brigadecore/brigade/releases/tag/v2.0.0-rc.1

Comment (krancour) [reply content/2021-11-18-brigade-2-cuts-first-release-candidate.md:12]: Thought about it, but if you follow that, it's a dead end because you'll get the CLI and that's it. If we link to the QuickStart, it'll walk them through everything.

Comment (krancour) [reply content/2021-11-18-brigade-2-cuts-first-release-candidate.md:12]: Actually... I could drop a link to the QuickStart on the release page and that helps...

Fixing...

Comment (krancour) [reply content/2021-11-18-brigade-2-cuts-first-release-candidate.md]: Done

File: content/2021-11-18-brigade-2-cuts-first-release-candidate.md
Comment (vdice) [inline content/2021-11-18-brigade-2-cuts-first-release-candidate.md:3]: Was it intended to date this back in time (yesterday, 11/18).  Or, should it be dated today (11/19)?

**Review by vdice — APPROVED** [review]
🙌 LGTM!

(Only one Q around blog date.)

File: content/2021-11-18-brigade-2-cuts-first-release-candidate.md
Comment (krancour) [reply content/2021-11-18-brigade-2-cuts-first-release-candidate.md:3]: @vdice amazing catch! Not intentional. I'm just an idiot.

Comment (krancour) [reply content/2021-11-18-brigade-2-cuts-first-release-candidate.md]: Fixed!

Comment (vdice) [reply content/2021-11-18-brigade-2-cuts-first-release-candidate.md]: :shipit:

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (bridgetkromhout) [inline content/2021-11-19-brigade-2-cuts-first-release-candidate.md:16]: I would say "just in time" when there's a "for something" after it. "Just on time" usually stands alone.

Comment (krancour) [reply content/2021-11-19-brigade-2-cuts-first-release-candidate.md:16]: sgtm... changing it.

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (bridgetkromhout) [inline content/2021-11-19-brigade-2-cuts-first-release-candidate.md:18]: Do readers know what a peripheral is? Do you want to say something like "peripherals (foo, bar, bat, etc)"?

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (bridgetkromhout) [inline content/2021-11-19-brigade-2-cuts-first-release-candidate.md:20]: I know that "chore" is a designation, but in this context it might be clearer to say:

Last, but certainly not least, we also want to thank [Jorge Arteiro](https://github.com/jorgearteiro) for recently starting to review PRs across all the Brigade repos.

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (krancour) [reply content/2021-11-19-brigade-2-cuts-first-release-candidate.md:18]: Am I wrong to assume it's a commonly understood word? The first usage here (a couple paragraphs up) says "gateway and other peripherals." I _think_ it's understood to be inclusive of gateways as well as integrations that are _not_ gateways...

We could start favoring the term "integration," but I do like that peripheral sounds like what it is-- something we built as an accessory of sorts.

lmk what you think.

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (krancour) [reply content/2021-11-19-brigade-2-cuts-first-release-candidate.md:20]: This is a tricky one...

Technically, maintainers should review everything. The idea of designating certain PRs as reviewable by non-maintainers is a recent thing we're trying to alleviate some bottlenecks. I think there is value in maintaining that distinction...

But I could be convinced otherwise.

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (krancour) [reply content/2021-11-19-brigade-2-cuts-first-release-candidate.md]: Fixed in https://github.com/brigadecore/blog/pull/20/commits/217c03db44289decdb7aaf47098caf1d37dd7bac

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (bridgetkromhout) [reply content/2021-11-19-brigade-2-cuts-first-release-candidate.md:18]: People may understand the general concept, but they would probably like a list of which items you're talking about specifically. Maybe navigating over to https://github.com/brigadecore/ and picking a few to highlight would be helpful in making sure people know which items to go look at?

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (bridgetkromhout) [reply content/2021-11-19-brigade-2-cuts-first-release-candidate.md:20]: maybe "Last, but certainly not least, we also want to thank Jorge Arteiro for recently starting to assist in PR review across all the Brigade repos."

I'm trying to get away from the word "chore" since you already used it once in this short blog post, and since to a casual reader it may seem like you're diminishing the work he's doing (which isn't the impression you want to give).

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (krancour) [reply content/2021-11-19-brigade-2-cuts-first-release-candidate.md:18]: I see what you're saying. I'm going to include an example or two to lend some clarity to how I'm using the term.

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (krancour) [reply content/2021-11-19-brigade-2-cuts-first-release-candidate.md:20]: Yes... I like that!

Comment (krancour) [reply content/2021-11-19-brigade-2-cuts-first-release-candidate.md]: Fixed in https://github.com/brigadecore/blog/commit/021e94694b4438507085c17e00f00f3a260c001e

File: content/2021-11-19-brigade-2-cuts-first-release-candidate.md
Comment (krancour) [reply content/2021-11-19-brigade-2-cuts-first-release-candidate.md:18]: Fixed in https://github.com/brigadecore/blog/commit/021e94694b4438507085c17e00f00f3a260c001e

**Pull request merged**
